tools/obscuroscan/static/javascript.js

```diff
@@ -0,0 +1,26 @@
+const eventClick = "click";
+const eventDomLoaded = "DOMContentLoaded";
+const idGetBlockHeadHeight = "getBlockHeadHeight";
+const idBlockHeadHeight = "blockHeadHeight";
+const pathBlockHeadHeight = "/blockheadheight/";
+
+const initialize = () => {
+    const getBlockHeadHeightButton = document.getElementById(idGetBlockHeadHeight);
+    const statusArea = document.getElementById(idBlockHeadHeight);
+
+    getBlockHeadHeightButton.addEventListener(eventClick, async () => {
+        const blockHeadHeightResp = await fetch(pathBlockHeadHeight);
+
+        if (isOk(blockHeadHeightResp)) {
+            statusArea.innerText = `Current block head height: ${await blockHeadHeightResp.text()}`;
```

Comment:
If this gets more and more sophisticated we should consider using react or an alternative framework imo. (Not now but as we start adding more features, I think it'll save us time.)